CHANGES.md

```diff
@@ -2,6 +2,7 @@
 
 #### next release (8.7.11)
 
+- Move release guide from README.md to RELEASE_GUIDE.md4
```

Comment:
There's a "4" at the end of the last file name that I think is unintentional.

Reply:
fixed thanks